subprojects/docs/docs.gradle

```diff
@@ -451,6 +453,30 @@ samples {
             }
             // TODO (donat) sanity checks probably don't do anything for samples located one level down in the gradle/kotlin directories
         }
+
+        compositeBuildsBasics {
+            sampleDirectory = file("src/snippets/compositeBuilds/basic")
```

Comment:
I think once we're happy with this, we should move it out of src/snippets and into src/samples.

Reply:
👍

Reply:
Done